ml/backend/ggml/quant.go

```diff
@@ -0,0 +1,518 @@
+package ggml
```

Comment:
I'm not sure if "quant" the right word (slang?) here? quantization or quantize might be better

Reply:
Also, this file has a lot of non-standard names like `quantize_state_impl` , shouldn't we clean that up?